qiskit/pulse/commands/parametric_pulses.py

```diff
@@ -0,0 +1,292 @@
+# -*- coding: utf-8 -*-
```

Comment:
How is these functions different than what's in pulse_lib.continuous?

Reply:
I believe it's because these will only send a dict of params to the backend, while in `pulse_lib.continuous` an entire list of samples will be generated

Reply:
but I think that's what the parametric one will do in the case when the backend doesn't support the pulse. Maybe we want to consider getting rid of pulse_lib.continuous

Reply:
These are classes which do not create samples. pulse_lib.continuous has functions which are sampled

Reply:
These implement `Command`s, and `pulse_lib` does not, it is a library for sampling common pulse shapes.